### PR TITLE
Improve custom card documentation

### PR DESCRIPTION
**Pull request by basbruss**

### CODE REVIEW

**Comment by sildehoop** [comment]
If you want you can add #386 ??

**Comment by basbruss** [comment]
@sildehoop  that's more related to the usage  of custom_cards instead of developing them right? I can try to improve the docs for the usage of custom cards

**Comment by sildehoop** [comment]
Yes I know but #384, yes maybe that is the solution

**Comment by basbruss** [comment]
@sildehoop I have created a new page for downloading custom_cards and referenced from the old place to the new page

**Review by sildehoop — DISMISSED** [review]

File: mkdocs.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ nav:
       - Download: "setup/download.md"
       - Installation: "setup/installation.md"
       - Configuration: "setup/configuration.md"
+      - Custom_cards: "setup/custom_cards.md"
```
Comment (basbruss) [inline mkdocs.yml:18]: @stokkie90 Is this the only change I need to do to create a new page?

Comment (basbruss) [inline mkdocs.yml:18]: @stokkie90 Is this the only change I need to do to create a new page?

Comment (stokkie90) [reply mkdocs.yml:18]: The key in this case is the Title. So i would change it to: `"Custom Cards": "setup/custom-cards.md"`

**Review by stokkie90 — APPROVED** [review]

**Pull request merged**